05_loops/main.go

```diff
@@ -43,28 +42,130 @@ it's also the most explicit.
 */
 
 func main() {
-	var i int = 0
-	for {
-		if i >= 10 {
-			break
-		}
-		fmt.Println(i)
-		i += 1
-	}
+	// var i int = 0
+	// for {
+	// 	if i >= 10 {
+	// 		break
+	// 	}
+	// 	fmt.Println(i)
+	// 	i += 1
+	// }
 
 	// TODO: Write a loop that prints the numbers from 0 to 100, inclusive.
+	// var j int = 0
+	// for {
+	// 	if j > 100 {
+	// 		break
+	// 	}
+	// 	fmt.Println(j)
+	// 	j += 1
+	// }
 
 	// TODO: Write a loop that prints the numbers from 100 to 0, inclusive.
+	// var k int = 100
+	// for {
+	// 	if k < 0 {
+	// 		break
+	// 	}
+	// 	fmt.Println(k)
+	// 	k -= 1
+	// }
 
 	// TODO: Write a loop that print the numbers from 0 to 100, inclusive, but
 	// only the even numbers.
+	// var l int = 0
+	// for {
+	// 	if l > 100{
+	// 		break
+	// 	}
+	// 	if l % 2 == 0 {
+	// 		fmt.Println(l)
+	// 		}
+	// 	l += 1
+	// }
 
 	// TODO: Write a loop that prints the first 100 odd numbers.
+	// var m int = 0
+	// for {
+	// 	if m > 100 {
+	// 		break
+	// 	}
+	// 	if m % 2 != 0 {
+	// 		fmt.Println(m)
+	// 	}
+	// 	m += 1
+	// }
```

Comment:
This is incorrect.

This prints the odd numbers from 0 to 100.

What I asked is print the _first 100 odd numbers_.

E.g. 
- if I asked "print the odd numbers from 1 to 5", then I expect "1, 3, 5"
- if I asked "print the first 5 odd numbers, then I expect "1, 3, 5, 7, 9"

Reply:
This one is the reason why I didn't finish all your suggested changes yesterday. But after a while of researching, I come to the solution just by contemplating.